heartbeat/Makefile.am

```diff
@@ -144,7 +144,8 @@ ocf_SCRIPTS	     =  AoEtarget		\
 			varnish			\
 			vmware			\
 			vsftpd			\
-			zabbixserver
+			zabbixserver		\
+			lvmlockd
```

Comment:
Please add lvmlockd at the right place in the list of agents to keep the list sorted alphabetically, thank you :)

Reply:
OK, thanks!